imports/collections/users_private.js

```diff
@@ -18,6 +18,29 @@ Meteor.methods({
         console.log(user);
     },
 
+    'users.editProfile'(userId, eMail, longName) {
+        if (Meteor.isServer) {
```

Comment:
Yea. Please ommit the isServer-Check.
Below your method you'ss find the method `'users.admin.changePassword'(userId, password1, password2)`- maybe you copied the isServer test code from there? But this method needs this test, as the used API `Accounts.setPassword` id a SERVER-ONLY method. In your case you use a simple MongeDB update. This is OK for client & server.